adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/AbstractVertxBasedCoapAdapter.java

```diff
@@ -633,46 +650,105 @@ protected void preShutdown() {
             context.respondWithCode(ResponseCode.NOT_ACCEPTABLE);
             return Future.succeededFuture(ResponseCode.NOT_ACCEPTABLE);
         } else {
+            final String gatewayId = authenticatedDevice != null
+                    && !device.getDeviceId().equals(authenticatedDevice.getDeviceId())
+                            ? authenticatedDevice.getDeviceId()
+                            : null;
 
             final Span currentSpan = TracingHelper
                     .buildChildSpan(tracer, context.getTracingContext(),
                             "upload " + endpoint.getCanonicalName())
                     .ignoreActiveSpan()
-                    .withTag(Tags.COMPONENT.getKey(), getTypeName())
-                    .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_CLIENT)
```

Comment:
is there a particular reason why you want to remove these tags?

Reply:
They are already included in the parent spawn. 
I recognized, that the "kind" changes from "server" to "client". And coap uses "http-uri".
So my impression was, that the tags are not consequent maintained. Therefore I adjusted them in the was, I thought it is useful.

Reply:
The *kind* changes from `server` to `client` because this new span is used to track the protocol adapter's (client) request to the AMQP Messaging Network to forward the message to a downstream consumer. The parent span has been created in response to a device's request to upload data (hence `server`).
The tags are not inherited, thus it makes sense to set them on each new span so that they can easily be found when searching for tag values. FMPOV you should therefore revert this change ...

Reply:
done.